README.rst

```diff
@@ -69,21 +69,51 @@ need to follow these steps:
 
       git clone https://github.com/ansys/pysimai
 
-#. `Install PDM <https://pdm-project.org/latest/#recommended-installation-method>`_
+#. `Install PDM <https://pdm-project.org/latest/#recommended-installation-method>`_. NB: If you are a Windows user, make sure that Python is installed on your system and it is added to the Path.
```

Comment:
```suggestion
#. `Install PDM <https://pdm-project.org/latest/#recommended-installation-method>`_.
```

I think we should assume developers have python properly installed on their system.

Reply:
Just saw this PR.
So if I'm not a windows user I don't need to have "Python is installed on your system and it is added to the Path" ?